### PR TITLE
feat: cors settings on custom domain

### PR DESCRIPTION
**Pull request by Benehiko**
## Related Issue or Design Document

Add a guide on setting CORS origins on the project custom domain

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Review by aeneasr — COMMENTED** [review]
The PR has a lot of changes unrelated to it - could you clean it up please to make it easier to review? :)

**Comment by Benehiko** [comment]
> The PR has a lot of changes unrelated to it - could you clean it up please to make it easier to review? :)

I think this was due to `npm run format`. It hasn't been formatted in a while it seems...

**Comment by Benehiko** [comment]
I have only edited this file with the added images
https://github.com/ory/docs/blob/07f6284465fae779249af5cc249867011056833c/docs/guides/custom-domains.mdx

**Comment by aeneasr** [comment]
Could you still reset it? Otherwise there will be tons of merge conflicts with other PRs :/

**Comment by Benehiko** [comment]
yes i will :)

File: docs/guides/custom-domains.mdx [hunk 1]
```diff
@@ -40,6 +40,21 @@ whatever you supplied in the first step. See the
...
+domain you add, in this example we have `auth.foobar.ory` set as our custom
+domain which will point to your Ory Cloud Project. It is important that you add
+CORS origins to your custom domain to **ensure Single Page Applications (SPAs)
+support**. We allow up to **50 origins**. If you plan on us ing server-side
+rendering or native applications, you can safely disable CORS. You can read more
+about [CORS at MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+
+:::note
+
+We do not allow setting the CORS origins to be `*` or `null`. You can, however,
```
Comment (aeneasr) [inline docs/guides/custom-domains.mdx:53]: localhost is also not allowed

**Review by aeneasr — CHANGES_REQUESTED** [review]
Just a minor thing

**Pull request merged**